modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/ContentElementResourceTest.java

```diff
@@ -92,6 +93,26 @@ public void testGetAssetLibraryContentElementsPageWithSortDouble()
 			EntityField.Type.DOUBLE, _getUnsafeTriConsumer());
 	}
 
+	@Test
+	public void testGetSiteContentElementsPageFilterByContentType()
```

Comment:
As far as I understand, this test is creating one element and applying a filtering to check that this element has been returned. In this case it is possible that the filtering is not working at all, just returning everything. It would be better to have multiple elements and checking that the element specified in the filter is returned.